Testing plugins need package.json

(node 10.10 on Windows)

```
cordova-paramedic: installing plugins
cordova-paramedic: installing C:\Projects\Cordova\cordova-paramedic\spec\testable-plugin --no-telemetry --no-update-notifier
Invalid Plugin! C:\Projects\Cordova\cordova-paramedic\spec\testable-plugin needs a valid package.json
Error: Failed to install plugin : C:\Projects\Cordova\cordova-paramedic\spec\testable-plugin --no-telemetry --no-update-notifier
Completed tests at 8:34:54 PM
Collecting logs for the devices.
It looks like there is no target to get logs from.
Uninstalling the app.
Error: Failed to install plugin : C:\Projects\Cordova\cordova-paramedic\spec\testable-plugin --no-telemetry --no-update-notifier
    at PluginsManager.installSinglePlugin (C:\Projects\Cordova\cordova-paramedic\lib\PluginsManager.js:85:15)
    at PluginsManager.installPlugins (C:\Projects\Cordova\cordova-paramedic\lib\PluginsManager.js:37:14)
    at ParamedicRunner.installPlugins (C:\Projects\Cordova\cordova-paramedic\lib\paramedic.js:158:25)
    at ParamedicRunner.prepareProjectToRunTests (C:\Projects\Cordova\cordova-paramedic\lib\paramedic.js:147:10)
    at C:\Projects\Cordova\cordova-paramedic\lib\paramedic.js:75:21
    at _fulfilled (C:\Projects\Cordova\cordova-paramedic\node_modules\q\q.js:854:54)
    at C:\Projects\Cordova\cordova-paramedic\node_modules\q\q.js:883:30
    at Promise.promise.promiseDispatch (C:\Projects\Cordova\cordova-paramedic\node_modules\q\q.js:816:13)
    at C:\Projects\Cordova\cordova-paramedic\node_modules\q\q.js:877:14
    at runSingle (C:\Projects\Cordova\cordova-paramedic\node_modules\q\q.js:137:13)
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! cordova-paramedic@0.5.0 test-ios: `node main.js --platform ios --plugin ./spec/testable-plugin/ --verbose`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the cordova-paramedic@0.5.0 test-ios script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.
```

Plugins in `/spec` are all missing a `package.json` file.

--- a/spec/build-fail-plugin/package.json
+++ b/spec/build-fail-plugin/package.json
@@ -1,0 +1,11 @@
+{
+  "name": "build-fail-plugin",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "author": "",
+  "license": ""
+}

--- a/spec/build-success-plugin/package.json
+++ b/spec/build-success-plugin/package.json
@@ -1,0 +1,11 @@
+{
+  "name": "build-success-plugin",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "author": "",
+  "license": ""
+}

--- a/spec/sh-read-input-plugin/package.json
+++ b/spec/sh-read-input-plugin/package.json
@@ -1,0 +1,11 @@
+{
+  "name": "sh-read-input-plugin",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "author": "",
+  "license": ""
+}

--- a/spec/testable-plugin/package.json
+++ b/spec/testable-plugin/package.json
@@ -1,0 +1,14 @@
+{
+  "name": "testable-plugin",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "directories": {
+    "test": "tests"
+  },
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "author": "",
+  "license": ""
+}

--- a/spec/testable-plugin/tests/package.json
+++ b/spec/testable-plugin/tests/package.json
@@ -1,0 +1,11 @@
+{
+  "name": "tests",
+  "version": "1.0.0",
+  "description": "",
+  "main": "tests.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "author": "",
+  "license": ""
+}

